lib/Sema/TypeCheckStorage.cpp

```diff
@@ -2689,7 +2689,8 @@ static VarDecl *synthesizePropertyWrapperProjectionVar(
 }
 
 static void typeCheckSynthesizedWrapperInitializer(VarDecl *wrappedVar,
-                                                   Expr *&initializer) {
+                                                   Expr *&initializer,
+                                                   bool contextualize = false) {
```

Comment:
Let's not default this new parameter, instead specify this directly in every use.